tracer/build/supported_versions.json

```diff
@@ -1108,5 +1108,22 @@
         "maxVersionTestedInclusive": null
       }
     ]
+  },
+  {
+    "integrationName": "Yarp",
+    "assemblyName": "Yarp.ReverseProxy",
+    "minAssemblyVersionInclusive": "1.1.0",
+    "maxAssemblyVersionInclusive": "2.65535.65535",
+    "packages": [
+      {
+        "name": "Yarp.ReverseProxy",
+        "minVersionAvailableInclusive": "1.0.0",
+        "minVersionSupportedInclusive": "1.1.0",
+        "minVersionTestedInclusive": "1.1.2",
```

Comment:
Not an issue, but just wondering why we don't test < 1.1.2? I assume the pass-through proxy APIs weren't available then?

Reply:
Thanks for catching, I'll update this to test the latest 1.0.* and 1.1.0 where this change was introduced

Reply:
Added 1.0.* and 1.1.0 to be tested in b59e6ef55c3e420920cc987aaa958666caffcb4b